tools/sleigh-lift/src/main.cpp

```diff
@@ -140,7 +140,7 @@ int main(int argc, char *argv[]) {
   }
   // Parse arguments
   const std::string action = argv[1];
-  const char *sla_file_path = argv[2];
+  const char *sla_file_name = argv[2];
```

Comment:
I think we should still let the user specify a path to search for compiled spec files, like before, but have it be optional. If that directory path is provided, search that path and error if not found.

Reply:
The start address is an optional positional argument right now and I'd rather not bring in GFlags. Also, `getopt` is a no-go since it'll make MSVC support harder.

I might try something dumber like allow a special `-f` argument at the beginning to allow the path to be specified.

Reply:
298ff76